deepmd/pt/train/training.py

```diff
@@ -1032,10 +1032,13 @@ def log_loss_valid(_task_key="Default"):
             writer.close()
         if self.enable_profiler or self.profiling:
             prof.stop()
+            log.info(
+                f"The profiling trace has been saved under {self.tensorboard_log_dir}"
+            )
```

Comment:
This message is for `enable_profiler`, but one may set `enable_profiler` to `False` and `profiling` to True.